kolibri/core/errorreports/tasks.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+from django.db import connection
+from django.http import JsonResponse
+
+from .models import ErrorReports
+from kolibri.core.discovery.utils.network.client import NetworkClient
+from kolibri.core.discovery.utils.network.errors import NetworkLocationConnectionFailure
+from kolibri.core.discovery.utils.network.errors import NetworkLocationResponseFailure
+from kolibri.core.discovery.utils.network.errors import NetworkLocationResponseTimeout
+from kolibri.core.tasks.decorators import register_task
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_SERVER_URL = "https://telemetry.learningequality.org"
+
+DEFAULT_PING_JOB_ID = "10"  # Unsure about this value
+
+client = NetworkClient(DEFAULT_SERVER_URL)
```

Comment:
lets use `requests` instead

`requests.post(...)`. See how the current statistics are being sent to the telemetry server [here](https://github.com/learningequality/kolibri/blob/476571f11b6e6a0f18c73549ee279ac7d0511398/kolibri/core/analytics/utils.py#L455)